packages/filecoin-actor-utils/src/utils/abi.ts

```diff
@@ -0,0 +1,14 @@
+import { JsonFragment } from '@ethersproject/abi'
+import { Fragment } from 'ethers/lib/utils'
+
+export type ABI = ReadonlyArray<Fragment | JsonFragment | string>
+
+/**
+ * Returns a hex representation of a base64 encoded cbor string
+ */
+
+export const cborToHex = (b64Cbor: string): string => {
```

Comment:
@navFooh this is where we need to fix the `.subarray` to fit the cbor spec [here](https://www.rfc-editor.org/rfc/rfc7049#section-2.1)